ccpp/driver/GFS_diagnostics.F90

```diff
@@ -215,6 +287,46 @@ subroutine GFS_externaldiag_populate (ExtDiag, Model, Statein, Stateout, Sfcprop
       ExtDiag(idx)%data(nb)%var2 => IntDiag(nb)%ulwsfc(:)
     enddo
 
+    idx = idx + 1
+    ExtDiag(idx)%axes = 2
+    ExtDiag(idx)%name = 'rad_swdnt'
```

Comment:
We are using "DSWRFI" for 'instantaneous surface downward shortwave flux' and "DSWRFtoa" for the averaged 'top of atmos downward shortwave flux'. Can we use "DSWRFItoa" as the field name for consistency?

Reply:
@gthompsnWRF wanted to use these names, but I agree that it would be more consistent to use the UFS-GFS names and not the WRF names. I'll change this.